execution-engine/contracts/hdac-system/pop/src/pop_impl/economy.rs

```diff
@@ -225,6 +225,42 @@ impl ContractClaim {
             runtime::put_key(&name, Key::Hash([0; 32]));
         }
     }
+
+    // prefix: "ld"
+    // ld_{last_distribute_time}
+    pub fn read_last_distribute() -> Result<u64> {
```

Comment:
It seems to be better to store this single value under `Key::Local` than under `named_key`.

Reply:
Yes it could be. But I'm not sure which would be better, so I selected the similar way of total supply.

Reply:
If we keep this way, in my opinion, whenever we need to store some value in the contract, we need to produce `read_xxx`, `write_xxx` function repeatedly and it seems not maintainable. What makes you not sure?